tracing-fmt/Cargo.toml

```
@@ -1,8 +1,22 @@
 [package]
 name = "tracing-fmt"
-version = "0.1.0"
+version = "0.0.1-alpha.1"
 authors = ["Eliza Weisman <eliza@buoyant.io>"]
 edition = "2018"
+license = "MIT"
+documentation = "https://docs.rs/tracing-fmt/0.0.1-alpha.1/tracing-fmt"
+description = """
+An experimental subscriber for tracing, a runtime diagnostics system.
```

What about something more like 
```suggestion
A (currently experimental) `tracing` subscriber that formats and logs trace data.
```